src/components/DiagramFrame/DiagramFrame.tsx

```diff
@@ -149,7 +151,7 @@ export const DiagramFrame: React.FC<DiagramFrameProps> = ({
               borderRadius: "10px"}}
           />
           <IconButton
-            icon="Tune"
+            icon={<Visibility />}
```

Comment:
Expect to be tune from material-outline

Reply:
New icon 
![image](https://user-images.githubusercontent.com/56001784/115912428-2a46e800-a424-11eb-96bc-3a41546f56a0.png)